lib/cc/cli/validate_config.rb

```diff
@@ -56,11 +50,36 @@ def display_issues
         validator.warnings.each do |warning|
           puts "#{colorize("WARNING", :yellow)}: #{warning}"
         end
+      end
 
-        validator.nested_warnings.each do |warning|
-          puts "#{colorize("WARNING in #{warning.field}", :yellow)}: #{warning.message}"
+      def require_json_or_yaml
+        if !filesystem.exist?(Config::YAMLAdapter::DEFAULT_PATH) && !filesystem.exist?(Config::JSONAdapter::DEFAULT_PATH)
+          puts NO_CONFIG_MESSAGE
+          exit 0
+        elsif filesystem.exist?(Config::YAMLAdapter::DEFAULT_PATH) && filesystem.exist?(Config::JSONAdapter::DEFAULT_PATH)
+          puts "#{colorize("ERROR", :red)}: #{TOO_MANY_CONFIG_MESSAGE}"
+          exit 1
```

Comment:
Do we want to exit hard here or emit a warning and continue validating with the config file with higher precedence?

From what I remember, we're going to document a config file precedence giving priority to `.codeclimate.json`.

Reply:
Fair enough: I had favored erroring hard since it's a state that really isn't supposed to exist & warnings aren't the most user-obvious feedback if there aren't errors. But you're right, so I've pushed a new commit addressing that.

Reply:
I think we also have an app UI change in the pipeline to show which config file (or in-app config) is used, so this should be more visible.